[2022-02-09 08:59:19,303] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: yelloTaXi_HW_data_ingestee_gcs_dag.bigquery_external_table_task scheduled__2020-08-01T23:00:00+00:00 [queued]>
[2022-02-09 08:59:19,322] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: yelloTaXi_HW_data_ingestee_gcs_dag.bigquery_external_table_task scheduled__2020-08-01T23:00:00+00:00 [queued]>
[2022-02-09 08:59:19,322] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-09 08:59:19,323] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-02-09 08:59:19,323] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-09 08:59:19,343] {taskinstance.py:1259} INFO - Executing <Task(BigQueryCreateExternalTableOperator): bigquery_external_table_task> on 2020-08-01 23:00:00+00:00
[2022-02-09 08:59:19,350] {standard_task_runner.py:52} INFO - Started process 220 to run task
[2022-02-09 08:59:19,356] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'yelloTaXi_HW_data_ingestee_gcs_dag', 'bigquery_external_table_task', 'scheduled__2020-08-01T23:00:00+00:00', '--job-id', '466', '--raw', '--subdir', 'DAGS_FOLDER/yelloTaXi_data_ingestee_gcs_dag.py', '--cfg-path', '/tmp/tmps_46fvv_', '--error-file', '/tmp/tmpjd_31p2i']
[2022-02-09 08:59:19,358] {standard_task_runner.py:77} INFO - Job 466: Subtask bigquery_external_table_task
[2022-02-09 08:59:19,469] {logging_mixin.py:109} INFO - Running <TaskInstance: yelloTaXi_HW_data_ingestee_gcs_dag.bigquery_external_table_task scheduled__2020-08-01T23:00:00+00:00 [running]> on host cc021fb126ed
[2022-02-09 08:59:19,522] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-02-09 08:59:19,554] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=yelloTaXi_HW_data_ingestee_gcs_dag
AIRFLOW_CTX_TASK_ID=bigquery_external_table_task
AIRFLOW_CTX_EXECUTION_DATE=2020-08-01T23:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2020-08-01T23:00:00+00:00
[2022-02-09 08:59:19,558] {crypto.py:82} WARNING - empty cryptography key - values will not be stored encrypted.
[2022-02-09 08:59:20,643] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=yelloTaXi_HW_data_ingestee_gcs_dag, task_id=bigquery_external_table_task, execution_date=20200801T230000, start_date=20220209T085919, end_date=20220209T085920
[2022-02-09 08:59:20,697] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-09 08:59:20,739] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-02-09 09:14:29,186] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: yelloTaXi_HW_data_ingestee_gcs_dag.bigquery_external_table_task scheduled__2020-08-01T23:00:00+00:00 [queued]>
[2022-02-09 09:14:29,200] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: yelloTaXi_HW_data_ingestee_gcs_dag.bigquery_external_table_task scheduled__2020-08-01T23:00:00+00:00 [queued]>
[2022-02-09 09:14:29,200] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-09 09:14:29,200] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-02-09 09:14:29,201] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-09 09:14:29,217] {taskinstance.py:1259} INFO - Executing <Task(BigQueryCreateExternalTableOperator): bigquery_external_table_task> on 2020-08-01 23:00:00+00:00
[2022-02-09 09:14:29,224] {standard_task_runner.py:52} INFO - Started process 279 to run task
[2022-02-09 09:14:29,231] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'yelloTaXi_HW_data_ingestee_gcs_dag', 'bigquery_external_table_task', 'scheduled__2020-08-01T23:00:00+00:00', '--job-id', '554', '--raw', '--subdir', 'DAGS_FOLDER/yelloTaXi_data_ingesteee_gcs_dag.py', '--cfg-path', '/tmp/tmpw4rs5x7h', '--error-file', '/tmp/tmpq_z979a3']
[2022-02-09 09:14:29,232] {standard_task_runner.py:77} INFO - Job 554: Subtask bigquery_external_table_task
[2022-02-09 09:14:29,315] {logging_mixin.py:109} INFO - Running <TaskInstance: yelloTaXi_HW_data_ingestee_gcs_dag.bigquery_external_table_task scheduled__2020-08-01T23:00:00+00:00 [running]> on host 75953ff4f3d5
[2022-02-09 09:14:29,369] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-02-09 09:14:29,392] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=yelloTaXi_HW_data_ingestee_gcs_dag
AIRFLOW_CTX_TASK_ID=bigquery_external_table_task
AIRFLOW_CTX_EXECUTION_DATE=2020-08-01T23:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2020-08-01T23:00:00+00:00
[2022-02-09 09:14:29,394] {crypto.py:82} WARNING - empty cryptography key - values will not be stored encrypted.
[2022-02-09 09:14:31,060] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=yelloTaXi_HW_data_ingestee_gcs_dag, task_id=bigquery_external_table_task, execution_date=20200801T230000, start_date=20220209T091429, end_date=20220209T091431
[2022-02-09 09:14:31,092] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-09 09:14:31,127] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
